staging/src/slime.io/slime/modules/lazyload/charts/templates/namespace-global-sidecar.yaml

```diff
@@ -101,19 +101,14 @@ spec:
               value: {{ default 18181 $gs.probePort | quote }}
             - name: LOG_LEVEL
               value: {{ default "info" $g.log.logLevel }}
-            - name: WORMHOLE_PORTS
```

Comment:
we'd retain the ability of global sidecar, right? then why we revert this?

imo what we should change is the to-global-sidecar envoyfilter part.

Reply:
parameter retains